core/src/main/scala/org/scalarelational/column/property/NotNull.scala

```diff
@@ -1,8 +0,0 @@
-package org.scalarelational.column.property
-
-/**
- * @author Matt Hicks <matt@outr.com>
- */
-object NotNull extends ColumnProperty {
```

Comment:
I'm ready to compromise, but one of the goals of this pull request was to prevent ``null`` values that can easily go unnoticed. How about introducing a property ``Nullable`` instead to explicitly allow ``null`` values?

> Finally, it gives you the ability to write condition checks in more complex forms for NOT NULL and NULL.

Could you elaborate please?